index.html

```diff
@@ -218,13 +219,12 @@ <h3 class="h3">Skills</h3>
 
             <h2 class="h2">Get Started</h2>
             <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-            <form action="" class="contact__form">
-                <input type="text" placeholder="your name" name="name" class="name">
-                <input type="email" placeholder="your email" name="email" class="email">
-                <input type="text" placeholder="Enter your message here..." name="message" id="" class="message">
-                <input type="button" value="Start Collaboration" class="buttons contact__button">
+            <form action="https://formspree.io/f/mrgdozvp" method="post" class="contact__form">
+                <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+                <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
```

Comment:
[OPTIONAL] Nice one adding the pattern attribute but emails should not be case-sensitive. This means it should not check if I input it in lowercase or uppercase. So far I input a valid email then it should work which is the work of the type attribute email. I did use lowercase and it was submitted. I will encourage you to take a second look at it.
### Check the warning message, the user will not even understand the format you are expecting so if you are going to continue with this pattern, you have to add the expected format message, it maybe through the placeholder.
![image](https://user-images.githubusercontent.com/86200356/199456741-1dbc8cd8-0839-4606-80fb-b7aff0b03baf.png)